app/server/routes/v1/post.js

```diff
@@ -49,7 +50,7 @@ const User = require("../../models/user.model");
  *                  $ref: '#/components/schemas/Community/properties/title'
  *      responses:
  *        '404':
- *          description: User not found.
+ *          description: User or community not found.
```

Comment:
shouldn't this be one or the other?

Reply:
I hadn't really thought of that when I made this change but you're right. I'll make the change.